### ISSUE TITLE
Encoding error of accented letters

### ISSUE THREAD

**Issue by llusetti** [jjb3rd/Freshdesk.Net#4]
Using ASCII encoding all accented letters (à,è,ò....) are replaced by ? in the ticket body and subject.
I suggest to change all encodings to UTF8 to solve this problem in the various Write functions.

**Comment by jjb3rd** [comment]
@llusetti I'd be happy to integrate a pull request :)  Not trying to be lazy (or a jerk), but I don't actually use this library, so to a large extent updates rely on the contributions from others.  I created this as part of a SaaS proof of concept that only needed a few core pieces of the API, so that's all that I implemented.  At the same time, I thought others might want this same level of functionality, so it was released on github and packaged up for NuGet...it's not as extensive as the MobiHelpSDK, but it's something.  Sorry for the delayed response, and for not jumping on implementing the fix, but like I said, if you or anyone else wants to do it, I'll happily merge a pull request and release it once it passes the unit tests.
